src/app/users/users-update/users-update.component.ts

```diff
@@ -104,7 +128,28 @@ export class UsersUpdateComponent implements OnInit {
     this.router.navigate([ '/users/profile', this.user.name ]);
   }
 
-  bindFile(event) {
+  previewFile(event) {
+    const preview = <HTMLImageElement>document.querySelector('.profile-upload-image');
```

Comment:
This is not a good way to change the image src in Angular.  Instead you should bind the value to a variable on the class, the html would be like `[src]="previewSrc"`